services/ui-src/src/utilityFunctions/mathFunctions.js

```diff
@@ -1,6 +1,6 @@
 export const roundValue = value => {
```

Comment:
this function is not needed.  the same thing can be accomplished inline without a conditional.

example:

Math.round(value).toFixed(currentPrecision)

where currentPrecision = 1, for our case

Reply:
The rounding function has been replaced with the suggested code.